Marlin/Marlin_main.cpp

```diff
@@ -4159,7 +4146,7 @@ inline void gcode_M42() {
     float  X_current = current_position[X_AXIS],
            Y_current = current_position[Y_AXIS];
 
-    #if ENABLED(Z_PROBE_SLED) || ENABLED(Z_PROBE_ALLEN_KEY)
+    #if ENABLED(Z_PROBE_ALLEN_KEY)
```

Comment:
I'm trying to imagine a sled deploying and stowing with each probe point. Can that actually work?